clusterloader2/testing/load/modules/reconcile-objects.yaml

```diff
@@ -75,7 +76,8 @@ steps:
       action: start
       checkIfPodsAreUpdated: {{$CHECK_IF_PODS_ARE_UPDATED}}
       labelSelector: group = load
-      operationTimeout: 15m
+      operationTimeout: {{$operationTimeout}}
+      exactOperationTimeout: true
```

Comment:
I think we could move `exactOperationTimeout` to be CL2 flag/env that will change how WaitForPods works for whole test instead of particular measurements.